pkgs/development/tools/build-managers/gup/build.nix.gup

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu
+if [ -n "${GUP_TARGET:-}" ]; then
+	gup --always
+fi
+curl -LSs -o "$1" https://raw.githubusercontent.com/timbertson/gup/master/nix/gup-python.nix
```

Comment:
So you combine `master` version of `build.nix` with latest-release source version? Isn't that going to break?